runelite-client/src/main/resources/net/runelite/client/plugins/skillcalculator/skill_crafting.json

```
@@ -367,12 +367,30 @@
       "name": "Topaz Amulet (U)",
       "xp": 80
     },
+    {
+      "level": 45,
+      "icon": 6328,
+      "name": "Snakeskin boots",
+      "xp": 30
+    },
     {
       "level": 46,
       "icon": 567,
       "name": "Unpowered Orb",
       "xp": 52.5
     },
+    {
+      "level": 47,
+      "icon": 6330,
+      "name": "Snakeskin vambraces",
+      "xp": 35
+    },
+    {
+      "level": 48,
+      "icon": 18727,
```

https://github.com/runelite/runelite/blob/master/runelite-api/src/main/java/net/runelite/api/ItemID.java#L3782

```suggestion
      "icon": 6326,
```